From 432e08de88a27313608cced27f133a65e8a56c52 Mon Sep 17 00:00:00 2001
From: Emil Velikov <emil.l.velikov@gmail.com>
Date: Wed, 27 Jan 2016 11:59:43 +0000
Subject: [PATCH 018/117] tests/kmstest: inverse the order of LDADD libraries

The utils library depends on libdrm. Flip the order, orderwise we might
error during link stage like below:

  CC       main.o
  CCLD     kmstest
/usr/bin/ld: ../../tests/util/.libs/libutil.a(libutil_la-kms.o):
undefined reference to symbol 'drmOpen'

Reported-by: Tom Stellard <thomas.stellard@amd.com>
Tested-by: Tom Stellard <thomas.stellard@amd.com>
Signed-off-by: Emil Velikov <emil.l.velikov@gmail.com>
---
 tests/kmstest/Makefile.am | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/tests/kmstest/Makefile.am b/tests/kmstest/Makefile.am
index 100662e..ced541b 100644
--- a/tests/kmstest/Makefile.am
+++ b/tests/kmstest/Makefile.am
@@ -17,9 +17,9 @@ kmstest_SOURCES = \
 	main.c
 
 kmstest_LDADD = \
-	$(top_builddir)/libdrm.la \
+	$(top_builddir)/tests/util/libutil.la \
 	$(top_builddir)/libkms/libkms.la \
-	$(top_builddir)/tests/util/libutil.la
+	$(top_builddir)/libdrm.la
 
 run: kmstest
 	./kmstest
-- 
2.7.4

